pkgs/applications/networking/gabutdm/default.nix

```
@@ -15,6 +14,7 @@
 , json-glib
 , qrencode
 , curl
+, aria
```

```suggestion
, aria2
```

I don't understand why there was an `aria = aria2;` in all-packages.nix, it doesn't make sense...